site/src/theme/colors.ts

```diff
@@ -6,7 +6,8 @@ import { getMetadataAsJSON } from "utils/metadata";
 // so you can just set this to true.
 export const experimentalTheme =
   typeof document !== "undefined" &&
-  getMetadataAsJSON("experiments")?.includes("dashboard_theme");
+  (getMetadataAsJSON<string[]>("experiments")?.includes("dashboard_theme") ??
+    false);
```

Comment:
Ugly, but had to do this to make sure that the code is always type `boolean` and not type `any`

Reply:
yeah I hate this lol. but we can fix it later.